src/Symfony/Component/Form/Tests/Util/StringUtilTest.php

```diff
@@ -34,7 +34,7 @@ public function testTrimUtf8Separators($hex)
         $binary = pack('H*', $hex);
 
         // Convert UCS-2BE to UTF-8
-        $symbol = mb_convert_encoding($binary, 'UTF-8', 'UCS-2BE');
+        $symbol = mb_convert_encoding((string) $binary, 'UTF-8', 'UCS-2BE');
```

Comment:
Do we need this change? `pack()` should always return a string, shouldn't it?

Reply:
It can return false on error. As I'm not sure if/how such an error is handled, I've prefered transtyping it too. But I can remove it if you think it's unnecessary.

Reply:
> it can return false on error

On PHP 8, we should get a `ValueError` instead. So on PHP 8.1 we can be certain that `$binary` is a string. So yes, please revert this line.

Reply:
Done.